File: frontend/Models/DTHETRATRUOC.cs
using System;

namespace QuanLyBar.Client.Models
{
    public class DTHETRATRUOC
    {
        public int? Id { get; set; }
        public string Name { get; set; }
        public string Note { get; set; }
        public int? DnhomthetratruocId { get; set; }
        public bool? Status { get; set; }
        public int? UsermodifiedId { get; set; }
        public DateTime? Timemodified { get; set; }
        public DateTime? Timecreated { get; set; }
        public int? UsercreatedId { get; set; }
        public string Khoa { get; set; }
        public DateTime? Ngayhethan { get; set; }
    }
}

File: backend/src/models/DTHETRATRUOC.ts
import { Model, DataTypes } from 'sequelize';
import db from '../config/database';

class DTHETRATRUOC extends Model {
    public id!: any;
    public name!: any;
    public note!: any;
    public dnhomthetratruocid!: any;
    public status!: any;
    public usermodifiedid!: any;
    public timemodified!: any;
    public timecreated!: any;
    public usercreatedid!: any;
    public khoa!: any;
    public ngayhethan!: any;
}

DTHETRATRUOC.init({
    ID: {
        type: DataTypes.INTEGER,
        primaryKey: true, autoIncrement: true,
    },
    NAME: {
        type: DataTypes.STRING,
        
    },
    NOTE: {
        type: DataTypes.STRING,
        
    },
    DNHOMTHETRATRUOCID: {
        type: DataTypes.INTEGER,
        
    },
    STATUS: {
        type: DataTypes.BOOLEAN,
        
    },
    USERMODIFIEDID: {
        type: DataTypes.INTEGER,
        
    },
    TIMEMODIFIED: {
        type: DataTypes.DATE,
        
    },
    TIMECREATED: {
        type: DataTypes.DATE,
        
    },
    USERCREATEDID: {
        type: DataTypes.INTEGER,
        
    },
    KHOA: {
        type: DataTypes.STRING,
        
    },
    NGAYHETHAN: {
        type: DataTypes.DATE,
        
    },
}, {
    sequelize: db.sequelize,
    modelName: 'DTHETRATRUOC',
    tableName: 'DTHETRATRUOC',
    timestamps: false
});

export default DTHETRATRUOC;
